SimPEG/PF/Magnetics.py

```diff
@@ -28,49 +28,29 @@ class MagneticIntegral(Problem.LinearProblem):
     def __init__(self, mesh, **kwargs):
         Problem.BaseProblem.__init__(self, mesh, **kwargs)
 
-    def fwr_ind(self, m):
+    def fields(self, m, **kwargs):
+        """
+            Magnetic field data
+        """
+        self.model = m
 
         if self.forwardOnly:
 
             # Compute the linear operation without forming the full dense G
-            fwr_d = self.Intrgl_Fwr_Op(m=m)
+            fwr_d = self.Intrgl_Fwr_Op(m=self.chiMap*m)
```

Comment:
here, you can call 
```
self.chi
```
and that will deal with the mapping (and will work if the user provides `chi` as an input to the forward problem in the case that they are not running an inversion)